src/tools/miri/tests/fail/intrinsics/unchecked_shl.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(unchecked_math)]
```

Comment:
Oh, we don't have any `pass` tests for unchecked shifts in Miri? We should fix that... *oops*

Reply:
Ah, we do, they just call the intrinsics directly rather than calling the exposed functions. Phew.^^